docs/schemas/2024-10/lpa.json

```
@@ -18,7 +18,7 @@
     },
     "status": {
       "type": "string",
-      "enum": ["in-progress", "statutory-waiting-period", "registered"]
+      "enum": ["draft", "in-progress", "statutory-waiting-period", "registered", "do-not-register", "expired", "cannot-register", "cancelled", "de-registered", "suspended"]
```

Probably shouldn't include `draft` here, since LPAs will be in-progress when they first arrive in the LPA Store. There'll never be a draft one in the store.
```suggestion
      "enum": ["in-progress", "statutory-waiting-period", "registered", "do-not-register", "expired", "cannot-register", "cancelled", "de-registered", "suspended"]
```